public_dropin_environments/java_codegen/dr_requirements.txt

```diff
@@ -1,3 +1,3 @@
 pandas<=2.0.3
 pyarrow>=0.14.1,<=14.0.1
-datarobot-drum==1.10.13
+datarobot-drum==1.10.15
```

Comment:
I noticed that until here you referred to `datarobot-drum==1.10.14`.
Then looking throughout your changes I notice that sometimes you use `1.10.14` and sometimes `1.10.15`.
Is it wrong?

Reply:
I worked on base images a couple days ago, when 1.10.15 was not yet released. So base images have drum 1.10.14.

And environments will have 1.10.15 now.